### DISCUSSION TITLE
Sysmon logs not showing in Hunt but in kibana

### DISCUSSION THREAD

**Discussion by OzzyKampha** [Security-Onion-Solutions/securityonion#1834] [Unsupported Versions] [answered]
Sysmon logs not showing up in hunt, but are showing up in kibana. Any idea why?

**Comment by dougburks** [comment, marked as answer]
Please provide more detailed information:

How are you sending the sysmon logs to Security Onion?

How are you viewing the sysmon logs in Kibana?

How are you attempting to view the sysmon logs in Hunt?

**Reply by jertel** [reply]
Possibly related to #1824.

**Reply by OzzyKampha** [reply]
Yes it is related to that
